### PR TITLE
Add new CubeSensors pack

### PR DESCRIPTION
**Pull request by Kami**
This pull request adds a new pack for the [CubeSensors](https://cubesensors.com/) service.

### CODE REVIEW

**Comment by jfryman** [comment]
NEAT!

File: packs/openhab/pack.yaml [hunk 1]
```diff
@@ -1,7 +1,12 @@
 ### Place information about your pack version here.
 ---
 name: openhab
-description: Integraition with OpenHAB
+description: Integration with OpenHAB
+keywords:
+  - openhab
```
Comment (jfryman) [inline packs/openhab/pack.yaml]: Heh. Thanks.

**Comment by manasdk** [comment]
Should we have an st2IoT repo where all these packs can live.

Separating infra management and IoT would be nice from the branding perspective. Also, we may find it easier to attract some IoT enthusiasts. Also, jump on marketing bandwagons :P

**Comment by Kami** [comment]
@manasdk I think this should be solved by better discovery mechanisms, including a proper website for the registry.

I planned to add some discovery capabilities to `packs` command and create a  simple registry website for a while now (that's also why I added `keywords` attribute to the pack metadata file). I just need to find some time over a weekend :P

**Comment by Heshuge** [comment]
My company is IoT, so I've been watching all this pretty closely.

**Pull request merged**